alfresco-transform-libreoffice/alfresco-transform-libreoffice/src/main/java/org/alfresco/transformer/executors/LibreOfficeJavaExecutor.java

```diff
@@ -127,6 +121,10 @@ public void call(File sourceFile, File targetFile, String... args)
 
     public void convert(File sourceFile, File targetFile)
     {
+        if (jodconverter == null)
+        {
+            System.out.println("NULL!!!!");
+        }
```

Comment:
TODO - should use logger with a clear message and not drop through !!

Reply:
Agreed, improving this area of code is out of scope of the PR as add additional checks or erroring may affect functionality.